doc/releases_and_migration/migration_guide_v280.rst

```diff
@@ -0,0 +1,58 @@
+.. _migration_guide_v280:
+
+Migration guide for the v2.8.0 of the nRF Connect SDK
+*****************************************************
+
+.. contents::
+   :local:
+   :depth: 2
+
+Starting with the v2.8.0 release of the `nRF Connect SDK`_, there is a new Sidewalk provisioning module.
+For an overview of changes, read the following sections.
+
+Reasons for change
+==================
+
+The new provisioning module enhances security by:
+
+* Moving non-volatile Sidewalk keys to secure key storage.
+* Protecting the manufacturing data partition from write operations.
+* Always using the Key-Value Store version of manufacturing data.
+
```

Comment:
Just thought of it now, but maybe this (L22-41) could be a separate section called, e.g., "Implementation details", or a bit more precise: "Security enhancements details"? "Reasons for change" seem to end with the above bullet points

```suggestion

Security enhancements details
=============================
```